lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -579,7 +579,7 @@ def add_shipping_address(post, payment, options = {})
 
       def add_source_owner(post, creditcard, options)
         post[:owner] = {}
-        post[:owner][:name] = creditcard.name if creditcard.name
+        post[:owner][:name] = creditcard&.name if creditcard&.name
```

Comment:
The issue for this is that creditcard can be a string so this would still cause an error

Reply:
Does the &. operator not guard against it since everything in Ruby is an object? Or am I getting that wrong?

Reply:
My bad, misunderstood that the safe access `&` only works on objects. This is fixed in the latest push!